contrib/build-push-ceph-container-imgs.sh

```diff
@@ -6,6 +6,20 @@ set -ex
 #############
 # VARIABLES #
 #############
+CI_CONTAINER=${CI_CONTAINER:=false}
+if ${CI_CONTAINER} ; then
+  # save the ceph branch, from the parent job
+  CEPH_BRANCH=${BRANCH}
+  # must be set by caller (perhaps another Jenkins build job)
+  # CONTAINER_REPO_HOSTNAME="quay.io"
```

Comment:
Should we do some validation or are we 100% sure these vars will be present if `CI_CONTAINER` is true?